### ISSUE TITLE
Add support for all Messager Attributes

### ISSUE THREAD

**Issue by ikbenben** [hex-digital/nuxt-intercom#25]
@josephgu95 i am looking to do a quick PR for the custom_launcher_selector messenger attribute as outlined in issue/5. However while at it, I thought I would add support for all the attributes defined at https://developers.intercom.com/installing-intercom/docs/javascript-api-attributes-objects#section-messenger-attributes and thought I'd discuss strategy with you first. 

I would assume these attributes would be managed in the `intercom` property in nuxt.config.js however would be would you prefer it on the same level or a sub object. 

for example, the first option would be like the following
```
intercom: {
  appID: '234sds',
  debug: true,
  customLauncherSelector: '#intercom-btn',
  alignment: 'left',
  ...
} 
```
while the second option would be:
```
intercom: {
  appID: '234sds',
  debug: true,
  config: {
    customLauncherSelector: '#intercom-btn',
    alignment: 'left',
   ...
  }
  ...
} 
```
The second option makes it easier from a code perspective.

The overall approach I was going to take was to parse those properties at the top of the `lib/plugin.js` file and then pass the config settings to the Intercom object as part of the settings parameter (where debug is passed in). Then the applicationSettings javascript can be updated with these attributes when the app_id  is set

Does this work for you?

**Comment by Jamiewarb** [comment]
Hey @ikbenben 

Thanks a lot for this. I think the config key in the second example makes a lot of sense, and I think it's a common pattern seen in other modules, such as the [Sentry module](https://sentry.nuxtjs.org/guide/setup/). So yep, that works for me! Cheers

**Comment by ikbenben** [comment]
@Jamiewarb i've done my changes and want to create a PR but I think I need create branch permission in order to do this. can you sort me out? thanks

**Comment by Jamiewarb** [comment]
Hey @ikbenben 
This is now released in 0.0.10. Let me know if there's any issues :)
